test/std-test.cc

```diff
@@ -126,6 +149,20 @@ TEST(std_test, variant) {
 
   volatile int i = 42;  // Test compile error before GCC 11 described in #3068.
   EXPECT_EQ(fmt::format("{}", i), "42");
+
+  using V2 = std::variant<std::monostate, throws_on_move_t>;
```

Comment:
This alias is not needed, just use the variant type below.

Reply:
Fixed